code/modules/mob/living/basic/space_fauna/hivebot/hivebot_behavior.dm

```diff
@@ -0,0 +1,72 @@
+/datum/ai_behavior/find_and_set/hive_partner
+
+/datum/ai_behavior/find_and_set/hive_partner/search_tactic(datum/ai_controller/controller, locate_path, search_range)
+	var/mob/living/living_pawn = controller.pawn
+	var/list/hive_partners = list()
+	for(var/mob/living/target in oview(10, living_pawn))
+		if(!istype(target, locate_path))
+			continue
+		if(target.stat == DEAD)
+			continue
+		hive_partners += target
+
+	if(length(hive_partners))
+		return pick(hive_partners)
+
+
+/datum/ai_behavior/relay_message
```

Comment:
Could you document what these behaviours are for with doc comments (`///` or `/* */` format)?